Feature/8

YumeAyasaki opened the pull request:
In this PR, I have:
- Add sass to the project...
- Create global styling, including variables, mixins (although there's nothing there), and some stylings for headings and text...
- Make a Test page to demo some of the changes...

Note: I think we should unify (?) more things, maybe size for phones and tablets (there're many phone sizes, I just go with the 390px width on Figma) and maybe make a mixin for that. And maybe some other mixins if other members want to use...
And what is PR, again...?

YumeAyasaki committed:
File create and minor edit

Create file for mixins and variables, rename globals and home (module) to scss and some minor edit in index.js and _app.js...

YumeAyasaki committed:
Color variables

Add color variables + include variables file and mixin file in global.scss...

YumeAyasaki committed:
Global styling

Add global styling (header, body, small, details) + Make page test (just for general global styling, not variables yet)...

YumeAyasaki committed:
Global styling(1)

Minor edit for test page...

YumeAyasaki committed:
Add color + Edit text font

Add accent light and rename Poppins to Lora...

YumeAyasaki committed:
Delete not-relevant file

Delete Header file...

PaulHo0501 commented on styles/globals.scss:
```diff
@@ -0,0 +1,68 @@
+// Note: Font weight - https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight
+// Note: Ask about bold and bolder
+
+h1 { // Lora 32px bold
```
Change these styling into mixins or class. This will reduce the amount of repeated code.

PaulHo0501 replied:
Add another comment here to request changes.

PaulHo0501 reviewed (commented):
Please check the comment. The purpose of a global stylesheet is not to style individual HTML elements but to make mixins and classes that can be imported and used everywhere, and also, to reduce the amount the code needed to achieve the styling

YumeAyasaki committed:
Mixins and classes

Add mixins for Lora font and change all elements (tag?) to class...

YumeAyasaki committed:
Edit test page

Note: Body, smallest and details are not include...

YumeAyasaki committed:
Finalize global styling

Body, smallest and details. Editing test page for future test and other things...

YumeAyasaki commented:
Update PR:
- Change global styling to class...
- Add a mixin for Lora font...
- Edit the Test page according to the new global styling...?

Note: I want to make a mixin for phone media query, do we use max-width or min-width for phone size...?

PaulHo0501 commented on styles/globals.scss:
```diff
@@ -1,39 +1,29 @@
-  font-family: "Lora";
-  font-weight: 700;
-  font-size: 24px;
+.sub-header-2 { // Lora 24px bold
```
Please rename classes using camelUpperCase (eg. subHeader, subHeader2, ...)

YumeAyasaki committed:
Rename camelUpperCase

Rename class + minor edit for test page...

YumeAyasaki committed:
Rename camelUpperCase (1)

Rename class + minor edit for test page...

YumeAyasaki commented:
Update PR:
- Change name class...

PaulHo0501 commented on yarn.lock:
```diff
@@ -1,1608 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Solve the conflict with develop branch

YumeAyasaki committed:
Merge branch 'develop' of https://github.com/MEOStudio/vuonhoasu.org into feature/8

PaulHo0501 commented:
LGTM
